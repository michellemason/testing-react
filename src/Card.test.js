import React from "react";
import { render } from '@testing-library/react';
import Card from "./Card";

it('should render', () => {
    render(<Card  />)
});

it('should match snapshot', () => {
    const {asFragment} = render(<Card caption='Photo by Richard Pasquarella on Unsplash' />)
    expect(asFragment()).toMatchSnapshot();
});