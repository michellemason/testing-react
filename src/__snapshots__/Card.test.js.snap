// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
